Half/test/spec/half_spec.rb

```diff
@@ -0,0 +1,19 @@
+require '../lib/half'
+
+describe Half do
+  context "if you create a new instance of class" do
+    it "the class exists" do
+      half = Half.new
+      expect(half.class).to eq Half
+    end
+  end
+
+  context "when passed an array" do
+    it "returns two arrays of half the size"
+    test1      = Half.new
+    array     = [2, 5, 6, 8]
+    expected  = test1.half(array)
+
+    expect(expected).to eq [[2, 5][6, 8]]
```

Comment:
@colins is the the expected output that you envisioned for the half method? I had copied down two arrays next to each other, which isn't something that cannot be saved to a variable.